### DISCUSSION TITLE
Spacing in lists

### DISCUSSION THREAD

**Discussion by Louson** [requarks/wiki#6464] [Help / Questions]
In lists, there is space between items like they are different paragraphs. Is that something expected ?

**Comment by NGPixel** [comment]
No, provide an example?

**Comment by Louson** [comment]
With this markdown code:

```
This is a default paragraph. This is a default paragraph. This is a default paragraph. This is a default paragraph. This is a default paragraph. This is a default paragraph. This is a default paragraph. Spacing is reduced.

- this is list
- this is list
- this is list
- this is list
- this is list
- this is list
- this is list
```

The result is:

![image](https://github.com/requarks/wiki/assets/1019543/4d05e4d5-24f7-4e43-81f8-450354dd3ba0)

The spacing between each item looks larger than for paragraphs.
